app/components/NotesCard.tsx

```diff
@@ -0,0 +1,102 @@
+import Button from "react-bootstrap/Button";
+import { useState } from "react";
+import { CrashNote } from "@/types/crashNote";
+import { FaCirclePlus } from "react-icons/fa6";
+import NotesModal from "./NotesModal";
+import { UPDATE_CRASH_NOTE, DELETE_CRASH_NOTE } from "@/queries/notes";
+import RelatedRecordTable from "./RelatedRecordTable";
+import { ColDataCardDef } from "@/types/types";
+import { formatDate } from "@/utils/formatters";
+import { useAuth0 } from "@auth0/auth0-react";
+
+interface NotesCardProps {
+  notes: CrashNote[];
+  crashPk: number;
+  onSaveCallback: () => Promise<void>;
+  refetch: () => Promise<void>;
+}
+
+const notesColumns: ColDataCardDef<CrashNote>[] = [
+  {
+    path: "date",
+    label: "Date",
+    editable: false,
+    valueFormatter: formatDate,
+  },
+  {
+    path: "user_email",
+    label: "User",
+    editable: false,
+  },
+  {
+    path: "text",
+    label: "Note",
+    editable: true,
+    editableCheck: (record: CrashNote, currentUserEmail?: string) =>
+      record.user_email === currentUserEmail,
+    inputType: "textarea",
+    style: { minWidth: "350px" },
+  },
+  {
+    path: "actions",
+    label: "",
+    editable: false,
+    style: { width: "50px" },
+  },
+];
+
+const AddNoteButton = (handleShow: () => void) => {
+  return (
+    <Button size="sm" variant="primary" onClick={handleShow}>
+      <FaCirclePlus className="me-2" />
+      Add Note
+    </Button>
+  );
+};
+
+const NotesCard = ({ notes, crashPk, onSaveCallback }: NotesCardProps) => {
```

Comment:
it might be best to name this `CrashNotesCard` to be consistent with how we've named other crash-related components.. ideally we could have this card do double-duty for the location notes feature, but i think we can make that decision when we get there. at first glance i don't think it would be too much refactoring to extend this work with the location notes page.